### PR TITLE
Visual indicator for the eraser button

### PR DESCRIPTION
**Pull request by Nopraz**
Usage of the mouse move event to make a div follow the cursor and pointer-events:none for clicking below the div

### CODE REVIEW

**Comment by AshtrayShetty** [comment]
Looks good!! But could you do one small favor by hiding the eraser square when the cursor goes out of the canvas?

File: js/canvas.js [hunk 1]
```diff
@@ -299,9 +312,13 @@ func_buttons.forEach(button=>{
 
         if(button['title']==='Eraser/Color Eraser' && button['id']==='btn-pressed'){
 
+            cursorEraser.style.display = "block";
+
             document.addEventListener('keydown', (e)=>{
                 if(e.keyCode===107){++width;}
                 if(width!==3 && e.keyCode===109){--width;}
+                cursorEraser.style.width = width + "px";
```
Comment (AshrayShetty4) [inline js/canvas.js:341]: Could you do one small favor by hiding the eraser square when the cursor goes out of the canvas?

**Review by AshrayShetty4 — CHANGES_REQUESTED** [review]

File: js/canvas.js (hunk 1 above)
Comment (Nopraz) [reply js/canvas.js:341]: Yeah sure

Comment (Nopraz) [reply js/canvas.js:341]: Just update the PR with commit c505ed4

Comment (AshrayShetty4) [reply js/canvas.js:341]: Sure thing!!! Thanks a ton!!

**Pull request merged**